bin/server/cmd/root.go

```diff
@@ -303,10 +313,44 @@ func NewMain(activeProfile Profile, logger *zap.Logger) (*Main, error) {
 			return nil, err
 		}
 	}
+
+	var db *store.DB
+	var pgStarted = false
+	if useEmbeddedDB() {
+		var err error
+		db, err = newEmbeddedDB(l, &activeProfile, pgInstance)
+		if err != nil {
+			return nil, err
+		}
+		pgStarted = true
+	} else {
+		var err error
+		db, err = newExternalDB(l, &activeProfile)
+		if err != nil {
+			return nil, err
+		}
+	}
+
+	// Open the connection of `bb` in Postgres and setup migration if needed
```

Comment:
'bb' is for prod. 'bbdev' for dev, 'bbtest' for test case. So we may not need to mention 'bb' explicitly or be specific that 'bb' is for prod.